enhancements/microshift/microshift-telemetry.md

```diff
@@ -0,0 +1,277 @@
+---
+title: microshift-telemetry
+authors:
+  - pacevedom
+reviewers:
+  - "ggiguash, MicroShift contributor"
+  - "pmtk, MicroShift contributor"
+  - "eslutsky, MicroShift contributor"
+  - "copejon, MicroShift contributor"
+approvers:
```

Comment:
this would need approval from the RHOBS team responsible for the Telemetry backend service.

/cc @moadz

Reply:
Just added to the list, thanks!